core/src/main/java/com/google/instrumentation/trace/TraceComponent.java

```diff
@@ -56,6 +56,14 @@
    */
   public abstract TraceExporter getTraceExporter();
 
+  /**
```

Comment:
The comment at line 20 should also mention this class (and Clock and TraceExporter)